frontend/src/tests/utils.js

```diff
@@ -289,6 +291,28 @@ export function checkPropTypes(propTypes, data) {
     }
 }
 
+/**
+ * parse spreadsheet file containing objects data into an array of objects
+ *
+ * @export
+ * @param {String} fileName
+ */
+export function parseSpreadsheet(fileName) {
+    const workbook = XLSX.readFile(
+        __dirname + `/import-export-data/${fileName}`
+    );
+    const sheet = workbook.Sheets[workbook.SheetNames[0]];
+    let dataCSV = XLSX.utils.sheet_to_json(sheet, { header: 1 });
+    // transform to array of objects
+    const keys = dataCSV.shift();
+    dataCSV = dataCSV.map(function (row) {
```

Comment:
I suggest using the same way to define anonymous functions inside array functions (map, some, foreach). It is more concise to use arrow functions

Reply:
For this code, I believe it's clearer to do in this way as I am constructing an object from an array which may be hard to understand if we make it compact using arrow functions. For another part below, I believe you are correct and changed to use arrow functions.